catroid/src/org/catrobat/catroid/content/bricks/AddItemToUserListBrick.java

```diff
@@ -49,14 +48,14 @@
 import org.catrobat.catroid.ui.adapter.DataAdapter;
 import org.catrobat.catroid.ui.adapter.UserListAdapterWrapper;
 import org.catrobat.catroid.ui.dialogs.NewDataDialog;
-import org.catrobat.catroid.ui.dialogs.NewDataDialog.NewUserListDialogListener;
 import org.catrobat.catroid.ui.fragment.FormulaEditorFragment;
 
 import java.util.List;
 
-public class AddItemToUserListBrick extends FormulaBrick implements NewUserListDialogListener {
+
```

Comment:
Darn, we missed that one :scream_cat: I'm just pointing it out because I would be really, really, really happy if you rebase again. Then I'll merge it immediately.
(There is an AndroidStudio formatter setting to define the max empty lines. We have to change it from 2 to 1. I'm just waiting for the best moment which would be a formatter run over all files (again :crying_cat_face:)